src/crimson/net/Socket.h

```diff
@@ -3,21 +3,26 @@
 
 #pragma once
 
+#include <seastar/core/gate.hh>
```

Comment:
the commit message reads

> we don't need to move them later in our application, which is not

i think i'd better to put

> we are not able to move them later in our application, which is not
> ...
> even though FixedCPUServerSocket is a sharded service,
> only the service located on the shard where the socket is originally created will be serving incoming connections. it's not allowed to perform i/o with that connected socket other than the original one.

Reply:
fixed